packages/clay-panel/src/index.tsx

```diff
@@ -146,9 +146,9 @@ function ClayPanel({
 								show: internalExpanded,
 							}
 						)}
+						data-disclosure
```

Comment:
What is the purpose of adding this data attribute?

Reply:
I removed it